ui/partials/main/thJobDetailsRetriggerMenu.html

```diff
@@ -1,6 +1,7 @@
 <li>
   <a ng-click="retriggerJob([selectedJob])">Retrigger job</a>
 </li>
-<li>
-  <a ng-click="backfillJob()">Backfill job</a>
+<li ng-class="currentRepo.repository_group.name == 'try' ? 'disabled' : ''"
+    ng-attr-title="{{currentRepo.repository_group.name == 'try' ? 'Backfilling not available in this repository' : ''}}">
+  <a ng-disabled="currentRepo.repository_group.name == 'try' ? 'disabled' : ''" ng-click="backfillJob()">Backfill job</a>
```

Comment:
There's quite a bit of duplication here - I'd add a new function called something like `canBackfill` (similar to https://github.com/mozilla/treeherder/blob/4238fca43655caef566f0af3d1ff6d76a6dea41f/ui/plugins/controller.js#L356-L359) that you then refer to like happens here:
https://github.com/mozilla/treeherder/blob/2a9dbefa49d26d8d0ddc7c402c571691b1679770/ui/plugins/pluginpanel.html#L105